clients/src/main/java/org/apache/kafka/common/utils/Sanitizer.java

```diff
@@ -24,14 +24,12 @@
 import javax.management.ObjectName;
 
 /**
- * Utility class for sanitizing/desanitizing/quoting values used in JMX metric names
- * or as ZooKeeper node name.
+ * Utility class for sanitizing/desanitizing/quoting values used in JMX metric names.
  * <p>
- * User principals and client-ids are URL-encoded using ({@link #sanitize(String)}
- * for use as ZooKeeper node names. User principals are URL-encoded in all metric
- * names as well. All other metric tags including client-id are quoted if they
- * contain special characters using {@link #jmxSanitize(String)} when
- * registering in JMX.
+ * User principals and client-ids are URL-encoded using ({@link #sanitize(String)}.
+ * User principals are URL-encoded in all metric names as well. All other metric tags 
```

Comment:
The `as well` in the 2nd sentence does not make much sense anymore. What about:
```
User principals are URL-encoded using ({@link #sanitize(String)} in all metric names. All other metric tags
including client-id are quoted if they contain special characters using 
{@link #jmxSanitize(String)} when registering in JMX.
```

Reply:
sure, thanks for your comments, addressed it.